newt/repo/repo.go

```diff
@@ -3,7 +3,7 @@
  * or more contributor license agreements.  See the NOTICE file
  * distributed with this work for additional information
  * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
+ * to you under the Apache License, newtutil.RepoVersion 2.0 (the
```

Comment:
`find/replace` gone wrong! ;-)

Reply:
That's pretty funny :).  Thanks, I have fixed it.